docs/antrea-network-policy.md

```diff
@@ -1145,18 +1080,172 @@ spec:
+  ClusterNetworkPolicy's `appliedTo` and `to`/`from` peers.
+- For the `childGroup` field, currently only one level of nesting is supported:
+  If a ClusterGroup has childGroups, it cannot be selected as a childGroup by other ClusterGroups.
+- ClusterGroup must exist before another ClusterGroup can select it by name as its childGroup.
+  A ClusterGroup cannot be deleted if it is referred to by other ClusterGroup as childGroup.
+  This restriction may be lifted in future releases.
+- At most one of `podSelector`, `serviceReference`, `ipBlock`, `ipBlocks` or `childGroups`
+  can be set for a ClusterGroup, i.e. a single ClusterGroup can either group workloads,
+  represent IP CIDRs or select other ClusterGroups. A parent ClusterGroup can select different
+  types of ClusterGroups (Pod/Service/CIDRs), but as mentioned above, it cannot select a
+  ClusterGroup that has childGroups itself.
+
+**spec**: The ClusterGroup `spec` has all the information needed to define a
+cluster-wide group.
+
+**podSelector**: Pods can be grouped cluster-wide using `podSelector`.
+If set with a `namespaceSelector`, all matching Pods from Namespaces selected
+by the `namespaceSelector` will be grouped.
+
+**namespaceSelector**: All Pods from Namespaces selected by the namespaceSelector
+will be grouped.
+If set with a `podSelector`, all matching Pods from Namespaces selected by the
+`namespaceSelector` will be grouped.
+
+**ipBlock**: This selects a particular IP CIDR range to allow as `ingress`
+"sources" or `egress` "destinations".
+A ClusterGroup with `ipBlock` referenced in an ACNP's `appliedTo` field will be
+ignored, and the policy will have no effect.
+For a same ClusterGroup, `ipBlock` and `ipBlocks` cannot be set concurrently.
+ipBlock will be deprecated for ipBlocks in future versions of ClusterGroup.
+
+**ipBlocks**: This selects a list of IP CIDR ranges to allow as `ingress`
+"sources" or `egress` "destinations".
+A ClusterGroup with `ipBlocks` referenced in an ACNP's `appliedTo` field will be
+ignored, and the policy will have no effect.
+For a same ClusterGroup, `ipBlock` and `ipBlocks` cannot be set concurrently.
+
+**serviceReference**: Pods that serve as the backend for the specified Service
+will be grouped. Services without selectors are currently not supported, and will
+be ignored if referred by `serviceReference` in a ClusterGroup.
+When ClusterGroups with `serviceReference` are used in ACNPs as `appliedTo` or
+`to`/`from` peers, no Service port information will be automatically assumed for
+traffic enforcement. `ServiceReference` is merely a mechanism to group Pods and
+ensure that a ClusterGroup stays in sync with the set of Pods selected by a given
+Service.
+
+**childGroups**: This selects existing ClusterGroups by name. The effective members
+of the "parent" ClusterGrup will be the union of all its childGroups' members.
+See the section above for restrictions.
+
+**status**: The ClusterGroup `status` field determines the overall realization
```

Comment:
it is quite strange that `spec` and `status` are at the same indentation level as the other subfields, with nothing to differentiate them

Reply:
Added indent